packages/cli/templates/app/app/pages/index.tsx

```diff
@@ -8,9 +8,34 @@ const Home = () => (
     </Head>
 
     <main>
+      <div className="logo">
+        <img src="/logo.png" alt="blitz icon" height="72" width="72" />
```

Comment:
Isn't this height/width redundant having the same styles in css already?

Reply:
fixed in my branch